test/README.md

```diff
@@ -0,0 +1,97 @@
+# Running Tests
+
+To run all tests, perform the following steps:
+
+## Startup a fresh iRODS server
+
+Use the iRODS testing environment to setup an iRODS server: https://github.com/irods/irods_testing_environment
+
+### Get the iRODS network information
+
+We need the network that the HTTP API is running on to have all pieces of the server
+communicate with each other.
+
+Following is an example of how to list all available docker networks with examples
+of how the output may appear:
+
+```console
+$ docker network ls
+NETWORK ID     NAME                              DRIVER    SCOPE
+d06849108e9c   bridge                            bridge    local
+371117285a13   host                              host      local
+fbf586b6459f   none                              null      local
+297992bf57a9   ubuntu-2204-postgres-14_default   bridge    local
+```
+
+In the example output, the desired network would be `ubuntu-2204-postgres-14_default`.
+
+The docker network used will be referred to as `<NETWORK>` in the following sections.
+
+## Startup the Keycloak image for testing
+
+See [Build the Testing Image](#build-the-testing-image) to build the image first.
+
+After which, run the following command to startup keycloak:
+
+```console
+$ docker run --rm --network <NETWORK> -p 8080:8080 irods-http-api-keycloak start-dev
+```
+
+### Get detailed network information
+Detailed information of the network will be required to properly configure the HTTP API.
+
+To get this information run the following command:
+```console
+$ docker network inspect <NETWORK>
+```
+
+## Startup the iRODS HTTP API server
+See the [iRODS HTTP API README](/README.md) on how to build the iRODS HTTP API runner image. As well as launching the container.
+
+Be sure to add the `--network <NETWORK>` flag, to have the container be able to communicate with the iRODS server, as well as the Keycloak server.
+
+An example of how the command may look is as follows:
+```console
+$ docker run -d --rm --name irods_http_api \
+>    --network <NETWORK> \
+>    -v /path/to/config/file:/config.json:ro \
+>    -p 9000:9000 \
+>    irods-http-api-runner
+```
+
+### Ensure the HTTP API can communicate with the iRODS server and the keycloak image.
+If the HTTP API cannot communicate with the Keycloak server, the server should not
```

Comment:
Consider replacing `the server` with the `the HTTP API`.

_I'm assuming that's what you're referring to._

Reply:
Done.